src/packages/tests/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -349,12 +326,8 @@ async function teardownScenario(state: ScenarioState) {
   // props might be missing if test errors out before they are set.
   if (state.db) {
     await Promise.resolve(
-      state.scenario.down
```

Comment:
Only called in one sqlite scenario with empty strings which I assumed was a typo